doc/pages/user/020-accessing_and_exploring_a_realm.md

```diff
@@ -1,3 +1,43 @@
 # Accessing and Exploring a Realm
 
-In Astarte, a Realm is a logical partition which holds a number of devices and an Authentication Realm 
+In Astarte, a Realm is a logical partition which holds a number of devices and an Authentication Realm.
+
+The [Astarte Dashboard](015-astarte_dahsboard) allows you to explore all resources of a given Realm,
+such as e.g. Interfaces, Triggers, Devices, Groups etc...
+
+## Device limit in a Realm
+While there is no limit on the number of Devices registered in a Realm, it is possible to impose an upper
+bound using the `<astarte base API URL>/realmmanagement/v1/<realm name>/config/devices` API.
+By default, such an upper bound is set to infinity. The limit can be retrieved, updated or removed
+(reverting back to its default value).
+
+### Setting a limit on the number of registered Devices in a Realm
+
+`PUT <astarte base API URL>/realmmanagement/v1/<realm name>/config/devices`
+
+The HTTP payload of the request must have the following format:
+
+```json
+{"data": {"limit": <value>}}
+```
+
+The value must be a positive integer. If more Devices are registered than
+the requested limit, the HTTP call will fail.
+
+### Fetching the limit on the number of registered Devices in a Realm
+
+`GET <astarte base API URL>/realmmanagement/v1/<realm name>/config/devices`
+
+The HTTP payload of the response will have the following format:
+
+```json
+{"data": {"limit": <value>}}
+```
+
+The value can be either a positive number or `"infinity"`.
+
+### Removing the limit on the number of registered Devices in a Realm
+
+`DELETE <astarte base API URL>/realmmanagement/v1/<realm name>/config/devices`
```

Comment:
I'm not a fan of using a `DELETE` here.
Here we're are unsetting a value, not removing the entity `device` from config.
I'd rather have a `PUT` or `PATCH` setting the limit to `null` which is consistent with other Astarte APIs